README.md

```diff
@@ -1,88 +1,145 @@
-### Current limitations
-1) For the moment: 720x1280 px resolution only. Very soon support for any 9:16 resolution (Android). More to come.
-2) VNC app apparently can have hick-ups....
-3) Teleporting from location to location takes the game to load images. Faster phones may handle it better. We are testing on low end specs (Redmi 5A for 75 bucks). We will likely add a parameter to set the delays inbetween teleports and screenshots.
-4) Sometimes mons do not get reported to the DB. We are in the process of debugging. It can help to remove the files in the hash-folder however.
-
-# Installation on Server/Desktop
-1) Install python according to docs for your platform.
-
-  Having installed python, do get yourself python pip and run
-  `pip install -r requirements.txt`
-
-  Depending on your OS, you probably need to install some more stuff.
-  E.g. Ubuntu/Debian requires you to run
-  ```bash
-  sudo apt update
-  sudo apt install tesseract-ocr python-opencv
-  ```
-2) Create a .csv with coords of gyms (one coord per line in format 'lat,lng'):
-  ```bash
-  lat1,lng1
-  lat2,lng2
-  ...etc
-  ```
-  You can also try to download the coords of gyms from your existing RM DB (downloadDBCords.py)
-3) Insert gym images into gym_img folder. Either manually (e.g. from Ingress) or
-  try the downloadfortimg.py
-4) Configure config.ini as needed. See `config.ini.example` for params and info
-5) Start TRM by calling `python startWalker.py`
-
-For performance it might be worth giving ocr_multitask a chance and/or calling
-startWalker.py with `-os` in one console and `-oo` in another.
-
-
-# Steps on Android device
-1) Install Pogo, droidVNC, RemoteGpsController, systemizer
-2) Systemize RGC (Oreo requires it to be systemized as a priv-app)
-3) Start VNC as noted above, start RGC
-
-# WE DO NOT GUARANTEE FOR THE ENTIRE THING TO BE RUNNING PERFECTLY FINE
-
-### Todos
-
- - Write tests
- - Write VNC app
- - Improve scans
- - Improve errorhandling
- - Support more/all resolutions
-
-# Discord
-For minor help, reporting resolutions (instructions on how to do so will be given sometime in the future), bugreports
-[Join the discord server](https://discord.gg/MC3vAH9)
+# Map'A'Droid
+![Python 2.7](https://img.shields.io/badge/python-2.7-blue.svg)  
+![MAD-Banner](https://raw.githubusercontent.com/OfficialWiddin/TheRaidMapper/patch-1/examples/banner_small_web.png)
+
+The Raid Mapper is a Raid scanner for Pokemon GO, based on Android devices and OCR.  
+
+<img src="https://raw.githubusercontent.com/Grennith/TheRaidMapper/master/examples/example_map.PNG" width="500" height="500">
+
+## Information
+*  [Discord](https://discord.gg/MC3vAH9) - For general support  
+*  [Github Issues](https://github.com/Grennith/TheRaidMapper/issues) - For reporting bugs (not for support!)  
+
+## Requirements  
+* [PogoAssets](https://github.com/ZeChrales/PogoAssets)  
+* Google or PTC account with minimum level 5 and no Raid-bosses in the dex  
+* Mobile with Android 8.0 (API 26) or higher
```

Comment:
RGC itself runs on Android 5.0 (SDK 21?) or higher now. It may be unstable on anything less than 7.1.2 tho - needs improvements